go/consensus/cometbft/light/service.go

```diff
@@ -161,6 +161,9 @@ func (c *client) worker() {
 		if err != nil {
 			return fmt.Errorf("failed to convert protobuff light block into cometbft light block: %w", err)
 		}
+		if tmLb.SignedHeader == nil {
```

Comment:
When is signed header empty?
Can we use `ValidateBasic` here?

Reply:
> Using `ValidateBasic` here should avoid possible problems in other components of the light client.

Agree. `SaveLightBlock` is ok implemented, as `Store` should be responsible for storing not for validation.